src/proto/endpoint_pb.js

```diff
@@ -233,9 +295,12 @@ proto.iroha.protocol.ToriiResponse.prototype.getErrOrCmdName = function() {
 };
 
 
-/** @param {string} value */
+/**
```

Comment:
Again, not sure if a full description is necessary. If this is an auto-generated code that'll be discarded, comments are useless. If it's a library code, comments would help.

Reply:
The comments were also auto-generated along with the code.